src/bin/cratesfyi.rs

```diff
@@ -115,6 +118,7 @@ pub fn main() {
                     .help("Name of the crate to delete"))))
```

Comment:
```suggestion
                    .required(true)
                    .help("Name of the crate to delete"))))
```

Not directly part of your PR, but would be nice to give an error here (I did in https://github.com/rust-lang/docs.rs/pull/538)

Reply:
Good call, done.